README.md

```diff
@@ -1,2 +1,45 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
```

Comment:
thinking about it they don't actually need NodeJS installed because it'll be installled on the docker image

Reply:
I have a feeling you need it to run the docker-chs-development chs-dev command /bin/chs-dev references npm.

Reply:
Kept node js